daft_dashboard/daft_dashboard/__init__.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from daft_dashboard import daft_dashboard as native
+from importlib import resources
+
+
+DAFT_DASHBOARD_ENV_NAME = "DAFT_DASHBOARD"
+DAFT_DASHBOARD_URL = "http://localhost:3238/api/queries"
+
+
+def launch(block: bool = False):
```

Comment:
user should be able to pass in a custom port

```py
dashboard.launch(port=3825)
```

Reply:
I was thinking of this too, but wanted to keep this off until a follow-up PR. Wanted to scope this PR down as much as possible first.

Reply:
I think this is something that we'll expose in a future revision to the Daft dashboard.